.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+language: python
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+install: pip install -e ".[dev]"
+script: py.test --cov-report=xml --cov-report=html --cov-report term-missing --cov=awsprofile
```

Comment:
Looks like there isn't a newline at the end of the file

Reply:
Fixed